www/components/shelf/shelf.js

```diff
@@ -123,6 +121,7 @@ class shelf extends LitElement {
   }
 
   render() {
+    console.log('render()', this.shelfList);
```

Comment:
is there a reason to keep this?

Reply:
> TODO
> remove console debugging after verifying with Netlify deploy preview

just giving everyone a chance to review first 👍

Reply:
:D